dspace-api/src/main/java/org/dspace/administer/CreateAdministrator.java

```diff
@@ -103,20 +126,20 @@ protected CreateAdministrator()
      *
      * @throws Exception if error
      */
-    protected void negotiateAdministratorDetails()
-        throws Exception {
+    protected void negotiateAdministratorDetails(CommandLine line)
+            throws Exception {
         Console console = System.console();
 
         System.out.println("Creating an initial administrator account");
 
-        boolean dataOK = false;
-
         String email = null;
```

Comment:
```
String email = line.getOptionValue('e');
String firstName = line.getOptionValue('f');
String lastName = line.getOptionValue('l');
```
I tested this patch giving all three options plus `-p` and got an EPerson with no email address.

Reply:
I tried my suggested change and got the result I expected.

Reply:
I did not understand this point. Even after demorgan's changes this will not work?

Reply:
@mwoodiupui . Please help me understand.

Reply:
Dear @mwoodiupui ,

I changed:

```String email = null;
String firstName = null;
String lastName = null;```

to:

```
String email = line.getOptionValue('e');
String firstName = line.getOptionValue('f');
String lastName = line.getOptionValue('l');
```
Is this what you meant?

Reply:
Yes, that is what I meant.  Thank you.